presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -187,8 +190,7 @@ private boolean isLowHighSupportedForType(Type type)
                 || type.equals(BIGINT)
                 || type.equals(REAL)
                 || type.equals(DOUBLE)
-                || type.equals(DATE)
-                || type.equals(TIMESTAMP)) {
```

Comment:
what happened to timestamp?

Reply:
It is not supported by the optimizer. I found it to be misleading, that's why i decided to remove it until we support it in the optimizer.